tests/react/utils/types.test.tsx

```diff
@@ -6,28 +6,16 @@ it('useHydrateAtoms should not allow invalid atom types when array is passed', (
   function Component() {
     const countAtom = atom(0)
     const activeAtom = atom(true)
-    // Adding @ts-ignore for typescript 3.8 support
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    // @ts-expect-error TS2769
     useHydrateAtoms([
-      [countAtom, 'foo'],
-      [activeAtom, 0],
+      [countAtom, 'foo' as any],
+      [activeAtom, 0 as any],
```

Comment:
This breaks the goal of the test.
We want to make sure that it raises an TS error.